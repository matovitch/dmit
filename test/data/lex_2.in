_
a
Z
foo
_bAr
_bAr
_42
__bAz5_8