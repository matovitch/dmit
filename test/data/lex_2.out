{"tokens" :[{"token":"START_OF_INPUT"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"WHITESPACE"},{"token":"IDENTIFIER"},{"token":"END_OF_INPUT"}],"offsets":[32,31,30,29,28,27,26,23,22,18,17,13,12,9,8,0,0]}